november-2017/docker.md

```diff
@@ -0,0 +1,14 @@
+### Released: MyBB 1.8 Dockerfile & Docker Compose Recipe
+
+With the recent announcements of the 1.10 branch we've also got some great news for both plugin developers and theme designers. In recent weeks, work has been underway to create an official Docker image for the MyBB 1.8 series. This work has been taken on by a community member who proposed such a solution and set to work on getting it done. Also provided within the repository are two example docker-compose.yml files, to be used with Docker Compose or Swarm. Docker is a technology for creating lightweight containers to manage and orchestrate software. It works across multiple platforms, and makes it easy to quickly get a live instance of the current MyBB version running with a variety of different configurations without requiring much effort to install database management systems and such on your machine. Docker is a brilliant tool for developers to quickly prototype ideas in a fresh, consistent environment. You can find out more information on Docker [here](https://docs.docker.com/get-started/).
+
+Developers can immediately download and start using the current MyBB 1.8.13 Docker image from our official Docker Hub registry account by running the following command:
```

Comment:
Maybe a good idea to link to the Docker Hub repository: https://hub.docker.com/r/mybb/mybb/